modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -8240,6 +8178,42 @@ private void _testPutCustomObjectEntryUnlinkNestedCustomObjectEntries(
 		}
 	}
 
+	private void _testPutCustomObjectEntryWithDuplicateExternalReferenceCode(
+			String objectDefinitionRESTContextPath)
```

Comment:
This parameter is called different to provide the same information in the `_testPatchCustomObjectEntryWithDuplicateExternalReferenceCode` method. Please unify with the same name!